example/builder/src/basic.dart

```diff
@@ -1,14 +1,12 @@
-// ignore_for_file: deprecated_member_use_from_same_package
 import 'package:over_react/over_react.dart';
 
+// ignore_for_file: uri_has_not_been_generated
```

Comment:
Why is this ignore being added? Wasn't the plan to remove these?

Reply:
Just a follow up on this - for this PR did we plan on leaving it and addressing this is part 2?

Reply:
Ah, nope, I'll remove these now